### ISSUE TITLE
no-fn-reference-in-iterator

### ISSUE THREAD

**Issue by GuoSirius** [sindresorhus/eslint-plugin-unicorn#698]
```js
Vue.filter('isImagePlayer', isImagePlayer)
Vue.filter('getPlayerTypeAliasName', getPlayerTypeAliasName)
Vue.filter('formatPlayerInformation', formatPlayerInformation)

```

check is not passed!!!

**Comment by fisker** [comment]
Should be fixed by https://github.com/sindresorhus/eslint-plugin-unicorn/pull/697
